### PR TITLE
feat: add Test4 launch article

### PR DESCRIPTION
**Pull request by Kouteki**
add Test4 launch article

### CODE REVIEW

File: posts/2024-07-17_test4-live/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### A Stable Environment for dApp Development
+
+Test4 is the first permanent and stable testnet for Gno.land, designed to avoid breaking changes. To help developers use it to build and test dApps, we've added Test4 to the [Faucet Hub](https://faucet.gno.land/); you're able to obtain test tokens and experiment without real consequences.
+
+We've also collaborated with Onbloc to update [Adena](https://www.adena.app/), which now supports Test4 out of the box. Onbloc has also provided [GnoScan](https://gnoscan.io/), a transaction explorer to aid in smart contract development and troubleshooting.
+
+### Namespaces
+
+Namespaces provide users with the exclusive capability to publish contracts under their designated namespaces, similar to GitHub's user and organization model. Or simply put, you'll be able to claim a username and use it as your on-chain identity. Learn more about it in the [gnoland docs](https://docs.gno.land/concepts/namespaces/).
```
Comment (leohhhn) [inline posts/2024-07-17_test4-live/README.md:42]:
```suggestion
Namespaces provide users with the exclusive capability to publish contracts under their designated namespaces, similar to GitHub's user and organization model. Or simply put, you'll be able to claim a username and use it as your on-chain identity. Learn more about it in the [gno.land docs](https://docs.gno.land/concepts/namespaces/).
```

File: posts/2024-07-17_test4-live/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## What's Next?
+
+We've launched Test4 a few weeks ahead of the schedule - we'll be heads down the next few weeks fixing bugs and patching thingsS. There's still a lot of work left before we get to a mainnet release candidate. That's where you come in: play around with Test4, break it, and open issues on [GitHub](https://github.com/gnolang/gno) to let us know what needs to be fixed.
```
Comment (leohhhn) [inline posts/2024-07-17_test4-live/README.md:46]: Typo: 
```suggestion
We've launched Test4 a few weeks ahead of the schedule - we'll be heads down the next few weeks fixing bugs and patching things. There's still a lot of work left before we get to a mainnet release candidate. That's where you come in: play around with Test4, break it, and open issues on [GitHub](https://github.com/gnolang/gno) to let us know what needs to be fixed.
```

File: posts/2024-07-17_test4-live/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Only together we'll be able to build gno.land the right way. So let's get started, gnomies!
```
Comment (leohhhn) [inline posts/2024-07-17_test4-live/README.md:48]: > gnomies

lol

Maybe switch to `gnomes` for a more official tone.

**Review by leohhhn — APPROVED** [review]
Few minor comments, otherwise good to go!

File: posts/2024-07-17_test4-live/README.md (hunk 3 above)
Comment (Kouteki) [reply posts/2024-07-17_test4-live/README.md:48]: That one's intentional :)

File: posts/2024-07-17_test4-live/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+- Core engineering team
+- DevX team
+- Onbloc
+
+Which brings us to the second major feature, the GovDAO.
+
+### GovDAO
+
+The first iteration of the GovDAO enables on-chain voting and validator management. The initial GovDAO members are gno.land team members, major contributors and teams already building dapps on gno.land. The first voting proposals will be to add additional 3rd party validators:
```
Comment (michelleellen) [inline posts/2024-07-17_test4-live/README.md:26]: I would highlight this is a The first iteration of the testing GovDAO. In my opinion, we want to highlight this is not the final GovDAO structure and is solely for testing purposes at the moment

File: posts/2024-07-17_test4-live/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+### Namespaces
+
+Namespaces provide users with the exclusive capability to publish contracts under their designated namespaces, similar to GitHub's user and organization model. Or simply put, you'll be able to claim a username and use it as your on-chain identity. Learn more about it in the [gno.land docs](https://docs.gno.land/concepts/namespaces/).
+
+## What's Next?
+
+We've launched Test4 a few weeks ahead of the schedule - we'll be heads down the next few weeks fixing bugs and patching things. There's still a lot of work left before we get to a mainnet release candidate. That's where you come in: play around with Test4, break it, and open issues on [GitHub](https://github.com/gnolang/gno) to let us know what needs to be fixed.
+
+Only together we'll be able to build gno.land the right way. So let's get started, gnomies!
```
Comment (michelleellen) [inline posts/2024-07-17_test4-live/README.md:48]: I would also maybe link to this contributing readme https://github.com/gnolang/gno/blob/5083bcb6f4410c9e2a40d35416690ffbf84f64ce/CONTRIBUTING.md#L4

Comment (Kouteki) [reply posts/2024-07-17_test4-live/README.md:48]: fixed in https://github.com/gnolang/blog/pull/103/commits/5a844c2a0d1a2fb116dcf8962e056163d1245738

File: posts/2024-07-17_test4-live/README.md (hunk 4 above)
Comment (Kouteki) [reply posts/2024-07-17_test4-live/README.md]: Fixed in https://github.com/gnolang/blog/pull/103/commits/5a844c2a0d1a2fb116dcf8962e056163d1245738

**Pull request merged**